package.json

```diff
@@ -135,11 +125,13 @@
     "nodemon": "^2.0.7",
     "npm-run-all": "^4.1.5",
     "parse-ms": "^2.0.0",
+    "protractor": "^7.0.0",
```

Comment:
Do we still use protractor in anywhere?

Reply:
I think you're right, but I found it here https://github.com/okta/okta-signin-widget/blob/master/Gruntfile.js#L362.

I think this reference is dead code however